src/source/Common/RequestInfo.c

```diff
@@ -299,6 +299,10 @@ SERVICE_CALL_RESULT getServiceCallResultFromHttpStatus(UINT32 httpStatus)
         case SERVICE_CALL_NOT_AUTHORIZED:
         case SERVICE_CALL_NOT_IMPLEMENTED:
         case SERVICE_CALL_INTERNAL_ERROR:
+        case SERVICE_CALL_REQUEST_TIMEOUT:
```

Comment:
This is not really related to this PR but we need this for the upcoming changes for the intermittent producer and this is in general a good correctness enhancement